pages/test-evaluate/evaluation-tools/index.pl.md

```diff
@@ -0,0 +1,136 @@
+
+permalink: /test-evaluate/tools/pl  # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+ref: /test-evaluate/tools/  # Do not change this
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group and projects names. Leave the Working Group and projects acronyms in English.
+footer: >
+  <p>Uwaga dotycząca opisu wideo: Wideo na tej stronie nie zawiera zsynchronizowanej audiodeskrypcji, ponieważ obrazy jedynie ilustrują dźwięk i nie dostarczają dodatkowych informacji. W tym przypadku audiodeskrypcja byłaby bardziej rozpraszająca niż przydatna dla większości osób, w tym osób, które nie widzą elementów wizualnych. Opis informacji wizualnych jest zintegrowany w transkrypcji tekstowej z opisem wizualnym („transkrypcja opisowa”).</p>
+   <p><strong>Data:</strong> Aktualizacja 28 kwietnia 2020.</p>
+   <p><strong>Redakcja:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Film opracowany przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>) przy wsparciu projektu <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide</a> finansowanego przez Komisję Europejską (KE) w ramach programu Horyzont 2020 (Umowa o dotację 822245).</p>
+---
+
+Narzędzia do oceny dostępności cyfrowej to programy lub usługi online, które pomagają określić, czy treść internetowa spełnia standardy dostępności.
+
+<div class="video-card" id="video-intro">
+  {% include video-player-data.html
+      video-id="tools-for-evaluating"
+      yt-id="bn1XJSjc_qM"
+  %}
+  <p><a href="#alternatives">Omówienie alternatywnych narzędzi oceny - film</a></p>
+</div>
+
+[Lista narzędzi oceny dostępności cyfrowej](/test-evaluate/tools/list/)
+:   Zawiera informacje na temat ponad 100 narzędzi. Możesz skorzystać z filtrów, aby zawęzić listę do typów narzędzi, które Cię interesują.
+
+[[Wybór narzędzi oceny dostępności cyfrowej]](/test-evaluate/tools/selecting/)
+:   Zawiera wskazówki dotyczące wyboru narzędzi. Opisuje cechy i funkcjonalność różnych typów narzędzi oceny oraz omawia kwestie, które należy wziąć pod uwagę w konkretnej sytuacji.
+
+Powiązane materiały przedstawiono w: [[Omówienie oceniania dostępności internetowej]](/test-evaluate/)
+
+<hr>
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+##  Omówienie alternatywnych narzędzi oceny {#alternatives}
+
+{% include excol.html type="middle" %}
+
+_Ten film jest również dostępny na serwerze W3C: Film: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+###  Transkrypcja tekstowa z opisem elementów wizualnych dla filmu Wybór i stosowanie narzędzi oceny {#transcript}
+
+Filmy zawierają podstawową animację ilustrującą to, co mówi głos spoza ekranu. Ludzie są reprezentowani przez postacie ikon.
```

Comment:
```suggestion
Filmy zawierają podstawową animację ilustrującą to, co mówi głos spoza ekranu. Ludzie są przedstawieni za pomocą symboli graficznych.
```
NOTES ON OIGINAL TEXT:
There's one video.
I have watched it and the second sentence does not make sense to me (People are represented by icon figures). If that sentence is necessary at all, it should read: "Concepts and ideas are represented by graphical symbols", or something similar.

Hence the text in Polish should rather be:
```suggestion
Film zawiera podstawową animację ilustrującą to, co mówi głos spoza ekranu. Pojęcia i idee są przedstawione w formie symboli graficznych.
```